src/schema/posts.ts

```diff
@@ -1702,59 +1623,6 @@ export const resolvers: IResolvers<any, Context> = {
       }
       return { _: true };
     },
-    upvote: async (
```

Comment:
We did double check nowhere used right? (just to be safe)

Reply:
Yep, frontend and extension use new votePost mutation for months now, also for cancel of upvote/downvote.

Reply:
Basically only someone on really old extension version.

Reply:
That's fine we given them enough time imo.

Reply:
Think we're good to go. Neither of these show up in metrics for the last 12 weeks
![image](https://github.com/dailydotdev/daily-api/assets/1681525/7565096e-db46-4cbe-8bbc-1895e8ac005c)